### ISSUE TITLE
Passive Diffusion Placenta Barrier

### ISSUE THREAD

**Issue by Chrizsly** [Open-Systems-Pharmacology/Pregnancy-Models#6]
Dear all,

Following the tutorial  (https://doi.org/10.1002/psp4.12300) on how to extend a generic PBPK model to include pregnant females, I encountered what I believe is a wrongful implementation of the equation describing the passive diffusion of a molecule across the placental barrier. More specifically, in MoBi, after loading a PK-Sim file and adding the spatial and passive transport building blocks, the equation that is implemented looks like this: 

![image](https://user-images.githubusercontent.com/57939062/91842621-a9e53400-ec54-11ea-8e4b-5e4849d60e8d.png)

However, the equation, like all other passive diffusion equations should look like this:
![image](https://user-images.githubusercontent.com/57939062/91842734-dc8f2c80-ec54-11ea-8c46-875f5114c8bf.png)

Basically, the fraction unbound is applied to both terms and the partition coefficient is divided rather than multiplied. 
I would appreciate any feedback on this very much. Thanks and best,

Christian

**Comment by msevestre** [comment]
@AndreDlm . Any comment on this?

**Comment by PavelBal** [comment]
The only problem that I see is that `K_cell_pls` should actually be `K_cell_wat` - so "Partitioning coefficient water/container" rather than "Plasma/container". Otherwise the original equation looks correct.

**Comment by PavelBal** [comment]
And since `Partition coefficient (intracellular/water)` is nothing else than `fu / K_cell_pls`, @Chrizsly is right.

**Comment by AndreDlm** [comment]
yes I confirm that

**Comment by Chrizsly** [comment]
Ok, thanks for the clarification. Would be good if this could be updated in the next version or so. Issue resolved now.

**Comment by msevestre** [comment]
Let's keep this one open since it's a bug;-)
